Branch-SDK-TestBed/build.gradle

```diff
@@ -1,13 +1,13 @@
 apply plugin: 'com.android.application'
 
 dependencies {
-    compile fileTree(dir: 'libs', include: '*.jar')
     compile project(':Branch-SDK')
-    debugCompile 'com.squareup.leakcanary:leakcanary-android:1.+'
-    releaseCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.3.1'
+    //debugCompile 'com.squareup.leakcanary:leakcanary-android:1.+'
+    //releaseCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.3.1'
```

Comment:
Yes. Since we are adding more dependency to SDK there is chance  for  dex over flow (Since we don't want to complicate test bed  with multi-dex implementation)  for test bed app. This is needed  only when we run memory leak test.